browser/static/js/main_zstack.js

```diff
@@ -51,144 +21,139 @@ class Mode {
 
     this.action = '';
     this.prompt = '';
-    adjuster.preCompAdjust(state.segArray, current_highlight, edit_mode, brush, this);
+    adjuster.preCompAdjust(canvas.segArray, current_highlight, edit_mode, brush, this);
   }
 
   // these keybinds apply regardless of
   // edit_mode, mode.action, or mode.kind
-  handle_universal_keybind(key) {
-    if (!rgb && (key === 'a' || key === 'ArrowLeft')) {
+  handle_universal_keybind(evt) {
+    if ((evt.ctrlKey || evt.metaKey) && evt.shiftKey && (evt.key === 'Z' || evt.key === 'z')) {
+      redo();
+    } else if ((evt.ctrlKey || evt.metaKey) && evt.key === 'z') {
```

Comment:
Hmmmm I buy that. I added the 'Z' or 'z' condition because just 'Z' wasn't working, but the Caps Lock issue is a good point. At some point, we should move to use [Mousetrap](https://craig.is/killing/mice)to handle the keybinds, which manages the case, Ctrl vs. Command for Windows/Mac, etc. For now I'll add the 'Z' or 'z' check to the second condition too.